### PR TITLE
Fix minor typing mistakes in docs

### PR DESCRIPTION
**Pull request by chutommy**
## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ PASTE HERE OR PROVIDE LINK TO https://pastebin.com/ OR SIMILAR
 
 <details><summary>System Details</summary>
 
-- OS (with version, e.g Windows 10 v2004 or macOS 10.15 (Catalina)):
+- OS (with version, e.g. Windows 10 v2004 or macOS 10.15 (Catalina)):
```
Comment (behackl) [inline .github/ISSUE_TEMPLATE/bug_report.md:57]:
```suggestion
- OS (with version, e.g., Windows 10 v2004 or macOS 10.15 (Catalina)):
```

File: .github/ISSUE_TEMPLATE/installation_issue.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ PASTE HERE OR PROVIDE LINK TO https://pastebin.com/ OR SIMILAR
 
 <details><summary>System Details</summary>
 
-- OS (with version, e.g Windows 10 v2004 or macOS 10.15 (Catalina)):
+- OS (with version, e.g. Windows 10 v2004 or macOS 10.15 (Catalina)):
```
Comment (behackl) [inline .github/ISSUE_TEMPLATE/installation_issue.md:40]:
```suggestion
- OS (with version, e.g., Windows 10 v2004 or macOS 10.15 (Catalina)):
```

**Review by behackl — APPROVED** [review]
Looks good to me in general, left two minor suggestions, please take a look. Thanks for contributing!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (Darylgolden) [reply .github/ISSUE_TEMPLATE/bug_report.md:57]: Putting a comma in front of e.g. seems weird to me. [The examples here](https://dictionary.cambridge.org/dictionary/english/eg) don't add a comma.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -182,7 +182,7 @@ Reports which involve higher risk or higher impact may face more severe conseque
 
 ### Propose consequences
 
-What follows are examples of possible consequences to an incident report. This consequences list is not inclusive, and the Manim Community Code of Conduct team reserves the right to take any action it deems necessary.
+What follows are examples of possible consequences of an incident report. These consequences list is not inclusive, and the Manim Community Code of Conduct team reserves the right to take any action it deems necessary.
```
Comment (Darylgolden) [inline CODE_OF_CONDUCT.md:185]: I think "This consequences list" is correct. There are multiple consequences, but the list is singular.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (behackl) [reply .github/ISSUE_TEMPLATE/bug_report.md:57]: AE vs BE: https://jakubmarian.com/comma-after-i-e-and-e-g/

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (Darylgolden) [reply CODE_OF_CONDUCT.md:185]: Also, I think "inclusive" should be changed to "exhaustive".

Comment (chutommy) [reply CODE_OF_CONDUCT.md:185]: > I think "This consequences list" is correct. There are multiple consequences, but the list is singular.

That's a good point. However shouldn't it be "This consequence list" instead? (if the word "consequence" serves as an adjective)

> Also, I think "inclusive" should be changed to "exhaustive".

I believe "**not** inclusive" is correct. Not all responses and actions are mentioned in the list.

Comment (Darylgolden) [reply CODE_OF_CONDUCT.md:185]: That SGTM.

**Review by Darylgolden — APPROVED** [review]
Thank you!

**Pull request merged**